snapshot-labs/sx-monorepo
# fix(ui): fix textarea overlapping over label on scroll

### Summary



Closes: #369

### How to test

1. Go to http://localhost:8080/#/create
2. Fill in the textarea with long info
3. Scrolling the textarea content should not overlap with the label anymore
